README.md

```diff
@@ -106,68 +78,135 @@ class ExampleTable extends Table
 You can use `SearchManager::add()` method to add filter or use specific methods
 like `value()`, `like()` etc. for in built filters.
 
-If you do not want to clutter your `initialize()` method with search config you
-can instead add a `searchManager()` method to the table class and return a search
-manager instance.
+### Filter collections
+
+The SearchManager has the ability to maintain multiple filter collections.
+For e.g. you can have separate collections for *backend* and *frontend*.
+
+All you need to do is:
 
 ```php
-class ExampleTable extends Table
-{
-    /**
-     * @param array $config
-     *
-     * @return void
-     */
-    public function initialize(array $config)
-    {
-        parent::initialize($config);
+// PostsTable::initialize()
+    $this->searchManager()
+        ->useCollection('backend')
+        ->add('q', 'Search.Like', [
+            'before' => true,
+            'after' => true,
+            'mode' => 'or',
+            'comparison' => 'LIKE',
+            'wildcardAny' => '*',
+            'wildcardOne' => '?',
+            'field' => ['body']
+        ])
+        ->useCollection('frontend')
+        ->value('name');
+```
 
-        // Add the behaviour to your table
-        $this->addBehavior('Search.Search');
+Let's use the *backend*'s filters by doing:
+
+```php
+// PostsController::action()
+    $query = $this->Examples
+        ->find('search', [
+            'search' => $this->request->getQuery(),
+            'collection' => 'backend',
+        ]);
     }
+```
 
-    /**
-     * @return \Search\Manager
-     */
-    public function searchManager()
-    {
-        $searchManager = $this->behaviors()->Search->searchManager();
-        $searchManager
-            ->like('title')
-            ->value('status');
+## Filter collection classes
+
+Apart from configuring filters through search mananger in your table class,
+you can also create them as separate collection classes. This helps in
+keeping your table's `initialize()` method uncluttered and the filters are lazy
+loaded only when actually used.
 
-        return $searchManager;
+```php
+// src/Model/Filter/Filter/PostsCollection.php
+namespace App\Model\Filter\PostsCollection;
```

Comment:
Is the namespace not just ...\Filter ?

Reply:
Thats my point: PostsCollection in the namespace then?
I expected `App\Model\Filter` only.

Reply:
Also the file name above does not match the namespace.